Failing test, maybe it could be removed?

Hi:

The test below is failing on my machine, but I am not sure what is its purpose. In my opinion, it could be removed. But maybe I am missing something.

```
xarray/tests/test_plot.py F
xarray/tests/test_plot.py:1027 (TestFacetGrid.test_no_args)
self = <xarray.tests.test_plot.TestFacetGrid testMethod=test_no_args>

    @pytest.mark.slow
    def test_no_args(self):
        self.g.map_dataarray(xplt.contourf, 'x', 'y')
    
        # Don't want colorbar labeled with 'None'
        alltxt = text_in_fig()
        self.assertNotIn('None', alltxt)
    
        for ax in self.g.axes.flat:
            self.assertTrue(ax.has_data())
    
            # default font size should be small
            fontsize = ax.title.get_size()
>           self.assertLessEqual(fontsize, 12)
E           AssertionError: 16.0 not less than or equal to 12

xarray/tests/test_plot.py:1041: AssertionError

```

Cheers

--- a/xarray/tests/test_plot.py
+++ b/xarray/tests/test_plot.py
@@ -1036,10 +1036,6 @@ class TestFacetGrid(PlotTestCase):
         for ax in self.g.axes.flat:
             self.assertTrue(ax.has_data())
 
-            # default font size should be small
-            fontsize = ax.title.get_size()
-            self.assertLessEqual(fontsize, 12)
-
     @pytest.mark.slow
     def test_names_appear_somewhere(self):
         self.darray.name = 'testvar'

